py-development.Rmd

```diff
@@ -407,8 +405,31 @@ packages right from your terminal, let's see how we can use them!
 
 ### Importing packages {#py-dev-importing-packages}
 
-Any installed package can be accessed by typing `import <package_name>` in
-Python, e.g.
+Many of the functions that we use regularly, such as `print()` and `len()` are 
+part of Python's Built-in functions. These are the functions that are always avaliable, 
+but often our programs will need tools beyond these built-ins. Importing in this 
+sense means that you are asking your program to "opt-in" to using that package's content.  
+The syntax for importing content is the same for stardand library packages, third 
+party packages, and local custom files. The lack of difference in syntax means that 
+code can become confusing if you aren't extremely conversant in many of the packages.  
+This is where good documentation comes in.
+
+Import statements are generally found at the beginning of script or notebook files.  
+Note that many of these examples will talk about importing functions, but there are 
+several other types of content that can be imported from a package. 
+
+There are two main ways of importing content:
+
+1. `import <package_name>`: Import the entire package, giving you access to the 
+package content by way of the package's name. E.g. `<package_name>.<function_name>()`.
+2. `from <package_name> import <function_name>`: Import the entire package, but only define variables for the functions you list explicitly.
+This gives you access to the imported content 
+without having to specify the package name. E.g. `function_name()`.
+
+Each of these methods has a way of renaming the package or content name to shorten 
+it or in case there are multiple with the same name and you need to avoid namespace clashes.
```

Comment:
```suggestion
it or in case there are multiple with the same name and you need to avoid namespace clashes. 
This is done using the `as` keyword, which is described in detail later in this chapter.
```